app/src/main/java/com/example/epamandroid/VkHomeActivity.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.epamandroid
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import android.support.v4.graphics.drawable.RoundedBitmapDrawable
+import android.support.v4.graphics.drawable.RoundedBitmapDrawableFactory
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.Toolbar
+import android.view.Menu
+import kotlinx.android.synthetic.main.activity_vk_home.activity_vk_home_toolbar
+import kotlinx.android.synthetic.main.vk_whatsnew_item.vk_whatsnew_item_user_image
+import kotlinx.android.synthetic.main.vk_photos_item.vk_photos_item_photos_count_textView
+import kotlinx.android.synthetic.main.vk_photos_item.vk_photos_item_photo_gallery_linearLayout
+
+class VkHomeActivity : AppCompatActivity() {
+
+    private lateinit var bitmap: Bitmap
+    private lateinit var roundedBitmapDrawable: RoundedBitmapDrawable
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_vk_home)
+
+        configureSupportActionBar()
+
+        createCircleImage()
+
+        vk_photos_item_photos_count_textView.text = vk_photos_item_photo_gallery_linearLayout.childCount.toString()
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.toolbar_menu, menu)
+
+        return super.onCreateOptionsMenu(menu)
+    }
+
+    private fun configureSupportActionBar() {
+        setSupportActionBar(activity_vk_home_toolbar as Toolbar?)
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        supportActionBar?.setDisplayShowHomeEnabled(true)
+        supportActionBar?.setTitle(R.string.name)
+    }
+
+    private fun createCircleImage() {
+        bitmap = BitmapFactory.decodeResource(resources, R.drawable.photo_vk)
+        roundedBitmapDrawable = RoundedBitmapDrawableFactory.create(resources, bitmap)
```

Comment:
better make like this 
![image](https://user-images.githubusercontent.com/16294951/54493529-3590c680-48e2-11e9-9342-1df5ddc568c4.png)

Reply:
@egorikftp done